src/cloud_common/protocols/unit/unit_config.py

```diff
@@ -233,7 +238,7 @@ class ResourceInfo(BaseModel):
         Field(
             default=None,
             alias='envs',
-            description='The list of environment variables.',
+            description='A list of environment variables.',
```

Comment:
The used in all other cases. So, keep here `the` as well.

Reply:
done